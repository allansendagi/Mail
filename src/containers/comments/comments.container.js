import React from 'react';
import {CommentsHeader} from "./comments-header/comments-header.component";
import { AddComment } from './add-comment/add-comment.container';
import { Comment } from './comment/comment.container';

export class Comments extends React.Component {
  render() {
    return(
      <div>
        <CommentsHeader amountComments={this.props.amountComments}/>
        <AddComment />
        <Comment />
        <Comment />
      </div>
    );
  }
}