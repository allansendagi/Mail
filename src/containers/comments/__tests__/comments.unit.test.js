import React from 'react';
import {shallow, configure} from 'enzyme';
import {Comments} from '../comments.container';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});
describe('Comments', () => {
  test('renders without props', () => {
    const wrapper = shallow(
      <Comments/>
    );
    expect(wrapper).toMatchSnapshot();
  });

  test('renders without amountComments', () => {
    const wrapper = shallow(
      <Comments amountComments={112499}/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
