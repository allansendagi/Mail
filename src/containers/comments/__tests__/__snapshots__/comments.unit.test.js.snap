// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments renders without amountComments 1`] = `ShallowWrapper {}`;

exports[`Comments renders without props 1`] = `ShallowWrapper {}`;
